app/assets/stylesheets/active_admin/mixins/_sections.scss

```diff
@@ -1,24 +1,16 @@
 @mixin section-header {
-  @include secondary-gradient;
-  @include text-shadow;
-  border: solid 1px #cdcdcd;
-  @include border-colors(#e6e6e6, #d4d4d4, #cdcdcd);
-  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 0 1px #FFF inset;
-
+  background-color: #e9e9e9;
```

Comment:
Color literals like `#e9e9e9` should only be used in variable declarations; they should be referred to via variable everywhere else.